docs/install/knative-with-operators.md

```diff
@@ -94,9 +120,59 @@ kind: KnativeServing
 metadata:
   name: knative-serving
   namespace: knative-serving
-EOF
+spec:
+  version: ma.mi.pa
+  manifests:
+    - URL: https://github.com/knative/serving/releases/download/v${VERSION}/serving-core.yaml
+    - URL: https://github.com/knative/serving/releases/download/v{VERSION}/serving-hpa.yaml
+    - URL: https://github.com/knative/serving/releases/download/v{VERSION}/serving-post-install-jobs.yaml
+    - URL: https://github.com/knative-sandbox/net-istio/releases/download/v0.17.0/net-istio.yaml
 ```
 
+The field `spec.version` is used to set the version of Knative Serving. Replace `ma.mi.pa` with the correct version number.
+The tag `${VERSION}` is automatically replaced with the version number from `spec.version` by the operator.
+
+The field `spec.manifests` is used to specify one or multiple URL links of Knative Serving component. Do not forget to
+add the valid URL of the Knative network ingress plugin. Knative Serving component is still tightly-coupled with a network
+ingress plugin in the operator. As in the above example, you can use `net-istio`. The list of URLs is order critical. Put
+the manifest you want to apply first on the top.
+
+{{< /tab >}}
+
+{{% tab name="Install Customized Knative Serving" %}} The operator provides you the flexibility to install customized
+Knative Serving based your own requirements. As long as the manifests of customized Knative Serving are accessible to
+the operator, they can be installed.
+
+For example, the version of the customized Knative Serving is 0.17.0, and it is available at `https://my-serving/serving.yaml`.
+You can create the content of Serving CR as below:
+
```

Comment:
I'd rewrite any of this to make it general, and if we can always just refer to a generic version or variable for it instead of saying specific number versions, e.g. v.0.17.0. It just saves maintenance / updates later.